acceptance/tests/sameness/sameness_test.go

```diff
@@ -0,0 +1,393 @@
+package sameness
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"testing"
+
+	terratestk8s "github.com/gruntwork-io/terratest/modules/k8s"
+	"github.com/hashicorp/consul-k8s/acceptance/framework/consul"
+	"github.com/hashicorp/consul-k8s/acceptance/framework/environment"
+	"github.com/hashicorp/consul-k8s/acceptance/framework/helpers"
+	"github.com/hashicorp/consul-k8s/acceptance/framework/k8s"
+	"github.com/hashicorp/consul-k8s/acceptance/framework/logger"
+	"github.com/hashicorp/consul/api"
+	"github.com/hashicorp/consul/sdk/testutil/retry"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	secondaryPartition    = "ap1"
+	primaryServer         = "dc1"
+	peer1                 = "dc2"
+	peer2                 = "dc2"
+	staticClientNamespace = "ns1"
+	staticServerNamespace = "ns2"
+
+	keyPrimaryServer = "server"
+	keyPartition     = "partition"
+	keyPeer1         = "peer1"
+	keyPeer2         = "peer2"
+
+	staticServerDeployment = "deploy/static-server"
+	staticClientDeployment = "deploy/static-client"
+
+	dnsLookup   = "static-server.service.ns2.ns.mine.sg.consul"
+	dnsPQLookup = "my-query.query.consul"
```

Comment:
IMO these should be defined as constants closer to where they get used so that it is an easier lookup instead of them being global constants.